### ISSUE TITLE
Integrate Samsung Proprietary(闭源驱动) driver

### ISSUE THREAD

**Issue by mshzhb** [longjunyu2/winlator#168]
We just brought up Samsung Proprietary driver for Exynos on proot linux.
Can we integrate into winlator? So winlator can support exynos chips.

https://github.com/user-attachments/assets/d38e8605-368a-499d-978b-200a4b0287d6

**Comment by lvonasek** [comment]
At the moment, we do not have capacity to do that. Here is a reference how it is build for Winlator (in case you want to build the patched driver yourself): https://github.com/K11MCH1/WinlatorTurnipDrivers

**Comment by mshzhb** [comment]
> At the moment, we do not have capacity to do that. Here is a reference how it is build for Winlator (in case you want to build the patched driver yourself): https://github.com/K11MCH1/WinlatorTurnipDrivers

Thanks for the feedback. Our driver is already compatible on proot linux (termux-x11). Does winlator use linux driver?

Thanks,
Tong

**Comment by lvonasek** [comment]
Yes, it does. It just require some specific patches to work.

**Comment by mshzhb** [comment]
Got it! I will check out these patches.

**Comment by daviDarthemis** [comment]
Winlator's x11 is different from the x11 we normally use in other projects, so modifications had to be made to the mesa-driver for it to work correctly.

**Comment by mshzhb** [comment]
> Winlator's x11 is different from the x11 we normally use in other projects, so modifications had to be made to the mesa-driver for it to work correctly.

Yes, our driver is already working with proot debian & termux-11
